kubernetes-model/cmd/generate/generate.go

```diff
@@ -135,10 +135,10 @@ type Schema struct {
 	RoleBindingRestriction                authapi.RoleBindingRestriction
 	LocalSubjectAccessReview              authapi.LocalSubjectAccessReview
 	SubjectAccessReview                   authapi.SubjectAccessReview
-        SubjectAccessReviewResponse           authapi.SubjectAccessReviewResponse
-	ClusterRole                           authapi.ClusterRole
-	ClusterRoleBinding                    authapi.ClusterRoleBinding
-	ClusterRoleBindingList                authapi.ClusterRoleBindingList
+  SubjectAccessReviewResponse          authapi.SubjectAccessReviewResponse
```

Comment:
Shouldn't we also prefix above RoleBinding classes from authapi with Openshift also?
```
Role            authapi.Role        -> OpenshiftRole                authapi.Role
RoleList      authapi.RoleList   -> OpenshiftRoleList          authapi.RoleList
....
```